server/test/e2e/shared-link.e2e-spec.ts

```diff
@@ -248,4 +258,81 @@ describe(`${PartnerController.name} (e2e)`, () => {
       expect(status).toBe(200);
     });
   });
+
+  describe('Shared link metadata', () => {
```

Comment:
I don't know how I feel about having a dependency on library asset scanning for this e2e test. I personally would prefer to keep e2e dependency on libraries to a minimum.

Reply:
To be honest, I've just copy-pasted the code from my library e2e tests since that was the easiest. Let me change this in another PR after this one, I'll refactor it to use an upload endpoint instead. (The format e2e tests have the same thing)

Reply:
Yeah that's fair. We can improve it in the future